sdk/tests/py/lint/lint.whitelist

```diff
@@ -0,0 +1,40 @@
+# File containing whiteslist for lint errors
+# Format is:
+# ERROR TYPE:file/name/pattern[:line number]
+# e.g.
+# TRAILING WHITESPACE:example/file.html:128
+# to allow trailing whitespace on example/file.html line 128
+
+## Whitespace rules that we can't enforce yet ##
+
+TRAILING WHITESPACE:*.md
+TRAILING WHITESPACE:*.txt
+
+## Accept executable permission ##
+
+UNNECESSARY EXECUTABLE PERMISSION:*.cgi
+UNNECESSARY EXECUTABLE PERMISSION:.git/*
+
+## Ignore INDENT TABS ##
+ 
+
```

Comment:
I'd suggest adding the following here, again since the original files contained these errors.

<pre>
INDENT TABS:*.vert
INDENT TABS:*.frag

# The original dEQP tests used tabs throughout.
INDENT TABS:deqp/functional/gles3/*.js
</pre>